implementations/rust/ockam/ockam_node/Cargo.toml

```diff
@@ -45,6 +45,7 @@ dump_internals = []
 [dependencies]
 ockam_core = { path = "../ockam_core", version = "^0.43.1-dev", default_features = false }
 ockam_macros = { path = "../ockam_macros", version = "^0.4.1-dev"}
+thread_local = ">=1.1.4"
```

Comment:
We don't need to add this dep to fix the [rust sec issue](https://github.com/ockam-network/ockam/runs/4930245013?check_suite_focus=true#step:7:22). A `cargo update` should update the dependency.